packages/arb-token-bridge-ui/src/hooks/useNetworksAndSigners.tsx

```diff
@@ -177,6 +178,9 @@ export function NetworksAndSignersProvider(
       const walletName = getWalletName(connector.name)
       trackEvent('Connect Wallet Click', { walletName })
     }
+
+    // set a custom tag in sentry to filter issues by connected wallet_name
+    Sentry.setTag('wallet_name', connector?.name ?? '')
```

Comment:
nit:
```suggestion
    Sentry.setTag('wallet.name', connector?.name ?? '')
```

I like how Sentry did it in their [documentation](https://docs.sentry.io/platforms/javascript/enriching-events/tags/), if we ever have something like `wallet.version` dots seem nicer than underscores

Reply:
Updated